src/I18n/TranslatorRegistry.php

```diff
@@ -43,6 +43,13 @@ class TranslatorRegistry extends TranslatorLocator
     protected $_defaultFormatter = 'default';
 
     /**
+     * Enable translation fallback
+     *
+     * @var bool
+     */
+    protected $_fallback = true;
```

Comment:
Naming it `$useFallback` would keep it consistent with the method name `I18n::useFallback()`.